File: Bai_02/src/nguyentrinhan70/atm/model/Account.java
package nguyentrinhan70.atm.model;

public class Account {
	private static int accountNo;
	private static String password;
	private static double amount;
	private String customerName;
	
	
	public Account() {
		super();
		accountNo =0;
		password ="";
		amount =0;
		customerName="";
	}
	public Account(int accountNo, String password, double amount, String customerName) {
		super();
		this.accountNo = accountNo;
		this.password = password;
		this.amount = amount;
		this.customerName = customerName;
	}
	public static int getAccountNo() {
		return accountNo;
	}
	public void setAccountNo(int accountNo) {
		this.accountNo = accountNo;
	}
	public static String getPassword() {
		return password;
	}
	public void setPassword(String password) {
		this.password = password;
	}
	public static double getAmount() {
		return amount;
	}
	public void setAmount(double amount) {
		this.amount = amount;
	}
	public String getCustomerName() {
		return customerName;
	}
	public void setCustomerName(String customerName) {
		this.customerName = customerName;
	}
	
	public boolean checkLogin(int accNo, String pass) {
		return(accNo==accountNo)&&(pass.equals(this.password));
		
	}
	
}

File: src_TH_05/models/Account.ts
export class Account {

  accountNo:string = '';
  password:string = '';
  amount:number = 0;
  
  constructor(){
    this.accountNo = '';
    this.password = '';
    this.amount = 100000;
  }

  setAccountNo(accountNo){
    this.accountNo = accountNo;
  }

  setPassword(password){
    this.password = password;
  }

  getAccountNo(){
    return this.accountNo;
  }

  getAmount(){
    return this.amount;
  }
}
